src/js/plugins/plugin.archive_analytics.js

```diff
@@ -53,7 +53,10 @@ BookReader.prototype.archiveAnalyticsSendFragmentChange = function() {
     archive_analytics.send_ping(values, null, "augment_for_ao_site");
 
     // Also send tracking event ping
-    archive_analytics.send_event('BookReader', 'UserChangedView', null);
+    var additionalEventParams = this.options.lendingInfo && this.options.lendingInfo.loanId
+      ? { loanId: this.options.lendingInfo.loanId }
+      : {};
```

Comment:
q: is this default an object because `archive_analytics.send_event` expects it to be one?

Reply:
Technically, anything false-y would work, but the default value it gets turned to is an object.